src/app/pages/storage/snapshots/snapshot-table/snapshot-list.component.html

```diff
@@ -0,0 +1,111 @@
+        {{ 'Delete' | translate }}
+      </button>
+    </div>
+
+    <div *ngIf="selection.selected.length !== dataSource.data.length" class="select-all" fxLayout="row" fxLayoutAlign="start center" fxLayoutGap="8px">
+      <span>{{ 'All {n} snapshots on this page are selected.' | translate: { n: selection.selected.length } }}</span>
+      <button mat-stroked-button (click)="selectAll()">{{ 'Select all {n} snapshots' | translate: { n: dataSource.data.length } }} </button>
+    </div>
+  </div>
+
+  <table
+    ix-table
+    [dataSource]="dataSource"
+    matSort
+    matSortDisableClear="true"
+    [matSortActive]="defaultSort.active"
+    [matSortDirection]="defaultSort.direction"
+    [multiTemplateDataRows]="true"
+    [fixedLayout]="true"
+  >
+    <ix-checkbox-column [selection]="selection" [dataSource]="dataSource"></ix-checkbox-column>
+
+    <ng-container matColumnDef="dataset">
+      <th mat-header-cell *matHeaderCellDef mat-sort-header> {{ 'Dataset' | translate }} </th>
+      <td mat-cell *matCellDef="let snapshot; dataSource: dataSource" [matTooltip]="snapshot.dataset">{{ snapshot.dataset }}</td>
+    </ng-container>
+
+    <ng-container matColumnDef="snapshot">
+      <th mat-header-cell *matHeaderCellDef mat-sort-header> {{ 'Snapshot' | translate }} </th>
+      <td mat-cell *matCellDef="let snapshot; dataSource: dataSource" [matTooltip]="snapshot.snapshot">{{ snapshot.snapshot }}</td>
+    </ng-container>
+
+    <ng-container matColumnDef="used">
+      <th mat-header-cell *matHeaderCellDef mat-sort-header> {{ 'Used' | translate }} </th>
+      <td mat-cell *matCellDef="let snapshot; dataSource: dataSource">{{
+        snapshot?.used ? (snapshot.used | convertBytestoHumanReadable) : ('N/A' | translate)
+      }}</td>
+    </ng-container>
+
+    <ng-container matColumnDef="created">
+      <th mat-header-cell *matHeaderCellDef mat-sort-header> {{ 'Date created' | translate }} </th>
+      <td mat-cell *matCellDef="let snapshot; dataSource: dataSource">{{
+        snapshot?.created ? (snapshot.created | formatDateTime) : ('N/A' | translate)
+      }}</td>
+    </ng-container>
+
+    <ng-container matColumnDef="referenced">
+      <th mat-header-cell *matHeaderCellDef mat-sort-header> {{ 'Referenced' | translate }} </th>
+      <td mat-cell *matCellDef="let snapshot; dataSource: dataSource">{{
+        snapshot?.referenced ? (snapshot?.referenced | convertBytestoHumanReadable) : ('N/A' | translate)
+      }}</td>
+    </ng-container>
+
+    <ng-container *ngIf="showExtraColumns$ | async; else expandRowButton" matColumnDef="actions">
+      <th mat-header-cell *matHeaderCellDef></th>
+      <td mat-cell *matCellDef="let snapshot; dataSource: dataSource">
+        <button
+          [title]="'Settings' | translate"
+          mat-icon-button
+          [matMenuTriggerFor]="actionsMenu"
+        >
+          <mat-icon>more_vert</mat-icon>
+        </button>
+        <mat-menu #actionsMenu="matMenu">
```

Comment:
An argument can be made that behaiour should be consistent between two views as not to be unexpected to the user.